src/runtime/runtime.cs

```diff
@@ -95,12 +95,18 @@ public class Runtime
 
         public static readonly string PythonDLL = _PythonDll;
 
-#if PYTHON_WITHOUT_ENABLE_SHARED && !NETSTANDARD
+        // In Unity, the library is always loaded in-memory
         internal const string _PythonDll = "__Internal";
-#else
-        internal const string _PythonDll = dllBase + dllWithPyDebug + dllWithPyMalloc;
+#if MONO_LINUX || MONO_MAC
+        internal const string dllDirectory = "Library/PythonInstall/lib/";
+        internal const string pythonlib = "python3.7m";
+// #elif MONO_MAC
+//         internal const string dllDirectory = "Library/PythonInstall/lib/";
+//         internal const string pythonlib = "python3.7m";
+#else //windows
+        internal const string dllDirectory = "Library/PythonInstall/";
+        internal const string pythonlib = "python37";
 #endif
```

Comment:
This is all very Unity specific. Maybe just allow `dllDirectory` to be set at build-time or something like that?